include/ginkgo/core/matrix/csr.hpp

```diff
@@ -162,6 +159,17 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
         int64_t clac_size(const int64_t nnz) { return 0; }
     };
 
+    class sparselib : public strategy_type {
```

Comment:
None of the strategies have any documentation. Seeing that they are actually public members and can ( need ) to be called, maybe it is a good idea to add documentation for all the strategy classes ? It would also be good to add some documentation for the constructors within the strategies as well.

Reply:
This comment also applies for the strategies for the hybrid class